packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -50,154 +50,166 @@ export const Disabled: ComponentStory<typeof FormFieldNext> = (props) => {
           to the input requirements.
-        </HelperText>
+        </FormHelperText>
       </FormFieldNext>
       <FormFieldNext labelPlacement="left" {...props}>
-        <Label>Default Form Field label</Label>
+        <FormLabel>Form Field label</FormLabel>
         <InputNext defaultValue="Primary Input value" />
-        <HelperText>
+        <FormHelperText>
           Helper text that's very long. Additional text to give further context
           to the input requirements.
-        </HelperText>
+        </FormHelperText>
       </FormFieldNext>
     </FlowLayout>
   );
 };
 
-/* TODO: These issues (in helper text) need consideration */
+/* TODO: These issues (in helper text) need consideration 
 
-export const MultiInput: ComponentStory<typeof FormFieldNext> = (props) => {
-  return (
-    <FlowLayout style={{ width: "366px" }}>
-      <FormFieldNext {...props}>
-        <Label>Paired fields</Label>
-        <Controls>
-          <InputNext variant="secondary" defaultValue="123" />
-          <InputNext variant="secondary" defaultValue="35" />
-        </Controls>
-        <HelperText>
-          *User entry in either field will automatically populate the
-          corresponding field with the correct value
-        </HelperText>
-      </FormFieldNext>
-      <FormFieldNext {...props}>
-        <Label>Multi criteria inputs</Label>
-        <Controls>
-          <InputNext variant="secondary" defaultValue="2.5" />
-          <InputNext variant="secondary" defaultValue="750" />
-        </Controls>
-        <HelperText>
-          *User must enter all values in the string to complete the input
-        </HelperText>
-      </FormFieldNext>
-    </FlowLayout>
-  );
-};
+Commenting out as it's possible but not supported until V3
+*/
+
+// export const MultiInput: ComponentStory<typeof FormFieldNext> = (props) => {
+//   return (
+//     <FlowLayout style={{ width: "366px" }}>
+//       <FormFieldNext {...props}>
+//         <FormLabel>Paired fields</FormLabel>
+//         <Controls>
+//           <InputNext variant="secondary" defaultValue="123" />
+//           <InputNext variant="secondary" defaultValue="35" />
+//         </Controls>
+//         <FormHelperText>
+//           *User entry in either field will automatically populate the
+//           corresponding field with the correct value
+//         </FormHelperText>
+//       </FormFieldNext>
+//       <FormFieldNext {...props}>
+//         <FormLabel>Multi criteria inputs</FormLabel>
+//         <Controls>
+//           <InputNext variant="secondary" defaultValue="2.5" />
+//           <InputNext variant="secondary" defaultValue="750" />
+//         </Controls>
+//         <FormHelperText>
+//           *User must enter all values in the string to complete the input
+//         </FormHelperText>
+//       </FormFieldNext>
+//     </FlowLayout>
+//   );
+// };
 
 export const Readonly: ComponentStory<typeof FormFieldNext> = (props) => {
```

Comment:
@libertymayc not sure if it's just storybook but i can make out a faint background in the secondary field when it should be transparent... can you see the same?

Reply:
fixed in #1666